### PR TITLE
chore: Remove imagenet

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Our ImageNet example does not work with the imagenet dataset after formatting changes. This dataset is also hosted by a 3rd party and requires a 3rd party account, adding extra steps outside of a simple download shell script. Let's remove this example as we have plenty of other computer vision examples. 

While this example is given the "porting" title, our MNIST tutorial also claims to be a "porting" title. Is it actually redundant?

## Test Plan

None

## Commentary (optional)

This example is not listed as a `keep` for examples-pruning, and this ticket was created in December 2022. Is there actual interest in this example?

## Checklist

- [X] Changes have been manually QA'd

## Ticket
MLG-400

### CODE REVIEW

**Comment by ioga** [comment]
I liked the "PyTorch Porting Tutorial" as a guide. I did not know it's dependent on the imagenet example. It's probably the best doc on "how to convert my code to Pytorch Trial API". Maybe we could keep it without referencing the broken example?..

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -23,10 +23,6 @@ To get started with your first experiment, visit the :ref:`Quickstart for Model
 |                                 | you how to port a simple image classification model for the  |
 |                                 | MNIST dataset.                                               |
 +---------------------------------+--------------------------------------------------------------+
-| :doc:`pytorch-porting-tutorial` | The PyTorch porting tutorial provides helpful hints to       |
-|                                 | successfully integrate PyTorch code with the Determined      |
-|                                 | PyTorchTrial API.                                            |
-+---------------------------------+--------------------------------------------------------------+
```
Comment (tara-det-ai) [inline docs/tutorials/index.rst:29]: if we are keeping the tutorial, we'll need to keep it in the index page. if we are removing the tutorial, we can remove it from the index page.

File: docs/tutorials/index.rst [hunk 2]
```diff
@@ -53,5 +49,4 @@ available.
 
    Run Your First Experiment <pytorch-mnist-local-qs>
    PyTorch MNIST Tutorial <pytorch-mnist-tutorial>
-   PyTorch Porting Tutorial <pytorch-porting-tutorial>
```
Comment (tara-det-ai) [inline docs/tutorials/index.rst:56]: same comment

**Review by tara-det-ai — COMMENTED** [review]
let's remove the PyTorch Porting Tutorial (which has less usage) in favor of improving the PyTorch MNIST Tutorial (which has more usage)

File: docs/tutorials/pytorch-porting-tutorial.rst
Comment (tara-det-ai) [inline docs/tutorials/pytorch-porting-tutorial.rst:1]: Let's remove this page. We'll need to remove it from the TOC tree on the index page.

File: docs/tutorials/index.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/tutorials/index.rst:29]: we'll also need to remove it from the toctree on this page.

**Review by tara-det-ai — COMMENTED** [review]
I suggest we remove the PyTorch Porting Tutorial.

**Review by tara-det-ai — APPROVED** [review]
LGTM

**Review by ioga — APPROVED** [review]

**Pull request merged**